frontend/libs/studio-components/src/components/StudioTextField/StudioIconTextfield/StudioIconTextfield.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StudioIconTextfield } from './StudioIconTextfield';
+import type { StudioIconTextfieldProps } from './StudioIconTextfield';
+
+describe('StudioIconTextfield', () => {
+  it('render the component', () => {
+    renderStudioIconTextfield({
+      icon: <div />,
```

Comment:
Should we expand this test to actually submit an icon and check if the icon indeed appears in the DOM itself? 😊

Reply:
yes, I will add it😊